### PR TITLE
chore: learning the PR process to understand Github's collaboration tools

### PR DESCRIPTION
**Pull request by itornaza**
I have just added the `constants.rs` file

### CODE REVIEW

File: src/constants.rs [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+const DEBUG: bool = false;
```
Comment (edilmedeiros) [inline src/constants.rs:2]: Why not call it `DEBUG_FLAG` instead?

**Review by edilmedeiros — CHANGES_REQUESTED** [review]
We better call `DEBUG_FLAG`.

File: src/constants.rs (hunk 1 above)
Comment (itornaza) [reply src/constants.rs:2]: No excuses! will do!

**Comment by itornaza** [comment]
Implemented suggestions!

**Review by edilmedeiros — APPROVED** [review]
ACK

**Pull request merged**